AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Operations/AWSAuthSignInOperation.swift

```diff
@@ -20,12 +20,15 @@ public class AWSAuthSignInOperation: AmplifySignInOperation,
                                      AuthSignInOperation {
 
     let authStateMachine: AuthStateMachine
+    let credentialStoreBehaviour: CredentialStoreStateBehaviour
 
     init(_ request: AuthSignInRequest,
          authStateMachine: AuthStateMachine,
+         credentialStoreClientFactory: CredentialStoreClientFactory,
          resultListener: ResultListener?) {
 
         self.authStateMachine = authStateMachine
+        self.credentialStoreBehaviour = credentialStoreClientFactory()
```

Comment:
Do we need this here? We are already passing this inside the environment. Can we fetch the device meta data inside the action and not pass it in SignInEventData?

Reply:
Oops.. Had it earlier for a different logic.. Removed it..

Reply:
This is used to fetch the device data used during sign in.

Reply:
Fixed.. removed it from here..